src/components/icons/clear-all-icon.js

```diff
@@ -1,4 +1,6 @@
-function clearAllIcon() {
-	return "<i class='icon fa-solid fa-c'></i>";
+import React from "react";
+
+export default function iconClearAll() {
```

Comment:
Follow the same way that you made for Button component

```js
<IconClearAll />
```

Reply:
Ok, But in button component you start with Capital letter

Reply:
hmmm a tested here, i didn't know this would work 🤔

Reply:
I know, I mean that you function name should be `IconClearAll`

Reply:
🤦‍♂️ i changed the name but deleted the change by accident and i didn't even noticed, next commit i change this again
it's right now

Reply:
done